src/ert/scheduler/lsf_driver.py

```diff
@@ -410,6 +411,50 @@ async def _process_job_update(self, job_id: str, new_state: AnyJob) -> None:
                 del self._iens2jobid[iens]
             await self.event_queue.put(event)
 
+    async def _get_exit_code(self, job_id: str) -> int:
+        process = await asyncio.create_subprocess_exec(
```

Comment:
You should probably use _execute_with_retry, because you want to retry on the 255 exit code

Reply:
uses `_execute_with_retry` now